Pipfile

```diff
@@ -14,6 +14,7 @@ requests = "*"
 numpy = "*"
 jsondiff = "*"
 requests-cache = "*"
+mock = "*"
```

Comment:
Should probably be under dev, right? (presumably true of pytest and the other testing packages as well)

Reply:
You're right. I thought the other packages were fixed, but they were only fixed in the `requirements.txt` and `requirements-dev.txt`